04-ruby-mvc-sinatra/sinatra-rest/readme.md

```diff
@@ -12,24 +12,43 @@
 - Use a global template - layout.erb
 - Use forms to submit POST and GET requests
 
+## Opening Framing
+We've learned about databases using SQL. We've also learned about objects in ruby. But we haven't seen any HTML since Unit 1! What gives? How are we going to display all of the things in our database? How can we create a UI to execute CRUD functionality on our different ruby objects? Sinatra. Today you'll be learning about how to create routes and views. In order to talk about how Sinatra functions, we need to talk about how HTTP requests work. Let's talk about REST.
+
 ## REST
 
-or, REpresentational State Transfer is a convention for making HTTP requests. We use REST to remove
-redundancy in network requests and agree on routes and what they do.
+or, REpresentational State Transfer is a software architectural style(convention) for making HTTP requests. We use REST to remove redundancy in network requests and agree on routes and what they do to create a more maintainable architecture.
+
+Every HTTP request consists of a request **method** and **path**
```

Comment:
kinda like punctuation marks, and who the recipient of a conversational transaction is